src/condor_scripts/cleanup_locally_mounted_checkpoint

```diff
@@ -29,6 +29,16 @@ if [[ $7 != "-delete" ]]; then
 fi
 DELETE=$8
 
+if [[ $9 != "-jobad" ]]; then
+    usage
+fi
+JOB_AD=${10}
+
+IGNORE_MISSING_FILES=
+if [[ ${11} == "-ignore-missing-files" ]]; then
+    IGNORE_MISSING_FILES=-f
+fi
+
```

Comment:
Do we need to add all these to usage?  also, looks like $JOB_AD is unused?

Reply:
Yeah, we should probably add it to the script's `usage()` function, since it has one.

The `${JOB_AD}` is deliberately present even though unused; I anticipate it will be necessary for S3.